exec/jdbc/src/main/java/org/apache/drill/jdbc/impl/DrillStatementImpl.java

```diff
@@ -159,24 +230,25 @@ public void cleanUp() {
   public int getQueryTimeout() throws AlreadyClosedSqlException
   {
     throwIfClosed();
-    return 0;  // (No no timeout.)
```

Comment:
it's an optional thing, but the spec doesn't say what happens if you set it but it's actually not used...